install/mlops/comet/install.yml

```
@@ -0,0 +1,14 @@
+id: comet-mlops
+name: Comet
```

This should be lower case
```suggestion
name: comet
```